airflow/providers/amazon/aws/triggers/redshift_cluster.py

```diff
@@ -80,8 +87,25 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:
                     else:
                         error_message = f"{self.task_id} failed"
                         yield TriggerEvent({"status": "error", "message": error_message})
+                elif self.operation_type == "delete_cluster":
+                    response = await hook.delete_cluster(
+                        cluster_identifier=self.cluster_identifier,
+                        skip_final_cluster_snapshot=self.skip_final_cluster_snapshot,
+                        final_cluster_snapshot_identifier=self.final_cluster_snapshot_identifier,
+                        poll_interval=self.poll_interval,
+                    )
+                    if response.get("status") == "success":
+                        yield TriggerEvent(response)
+                    else:
+                        if self.attempts < 1:
+                            error_message = f"{self.task_id} failed"
+                            yield TriggerEvent({"status": "error", "message": error_message})
                 else:
                     yield TriggerEvent(f"{self.operation_type} is not supported")
+                self.log.info(
+                    "Retrying in #%s seconds remaining attempts #%s", self.poll_interval, self.attempts
+                )
+                await asyncio.sleep(self.poll_interval)
             except Exception as e:
                 if self.attempts < 1:
                     yield TriggerEvent({"status": "error", "message": str(e)})
```

Comment:
This is still not correct I think. If one of the operations from the `try` block above raise an exception, it will get caught over here. Then depending on the number of attempts left (which may be more 1) a `TriggerEvent` will be yielded. Otherwise the exception will be silenced.

Reply:
this changes has been removed form this PR